packages/web-components/src/components/masthead/masthead-l1.scss

```diff
@@ -6,4 +6,5 @@
 //
 
 @import '@carbon/ibmdotcom-styles/scss/globals/vars';
-@import '@carbon/ibmdotcom-styles/scss/components/masthead/masthead-l1';
+// @import '@carbon/ibmdotcom-styles/scss/components/masthead/masthead-l1';
+@import '../../../../styles/scss/components/masthead/masthead-l1';
```

Comment:
Todo: Switch back to the import from the `@carbon/ibmdotcom-styles` once this PR is at a good place/ready to merge.